File: lista de exercicios/questão 1/questao.go
package main

import "fmt"

type Carro struct {
    Marca  string
    Modelo string
    Ano    int
}

func (c Carro) ExibirDetalhes() {
    fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", c.Marca, c.Modelo, c.Ano)
}

func main() {
    carro1 := Carro{Marca: "MarcaA", Modelo: "ModeloX", Ano: 2020}
    carro2 := Carro{Marca: "MarcaB", Modelo: "ModeloY", Ano: 2021}
    carro3 := Carro{Marca: "MarcaC", Modelo: "ModeloZ", Ano: 2022}

    carro1.ExibirDetalhes()
    carro2.ExibirDetalhes()
    carro3.ExibirDetalhes()
}

File: lista de exercicios/questão 1/questao.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir_detalhes(self):
        print(f"Marca: {self.marca}, Modelo: {self.modelo}, Ano: {self.ano}")

carro1 = Carro("MarcaA", "ModeloX", 2020)
carro2 = Carro("MarcaB", "ModeloY", 2021)
carro3 = Carro("MarcaC", "ModeloZ", 2022)

carro1.exibir_detalhes()
carro2.exibir_detalhes()
carro3.exibir_detalhes()
